frontend/review/src/components/ScoreBox.js

```diff
@@ -392,8 +409,12 @@ class ScoreBox extends Component {
   }
 
   componentDidUpdate(prevProps) {
-    const { data, liveData } = this.props;
-    if (prevProps.data !== data || prevProps.liveData !== liveData) {
+    const { data, liveData, isCourseEval } = this.props;
+    if (
+      prevProps.data !== data ||
+      prevProps.liveData !== liveData ||
+      prevProps.isCourseEval !== isCourseEval
```

Comment:
would prefer destructuring prevProps instead of accessing it everywhere in this function